docs/resources/v2-conformance-statement.md

```diff
@@ -309,6 +380,8 @@ The following `Accept` header(s) are supported for searching:
 
 - `application/dicom+json`
 
+Note that any instances stored with leniency applied to indexed attributes may not be retrievable by attributes which failed validation.
```

Comment:
Similar to the above comment, the reader might not know what we mean by "leniency" in this context.  Being more explicit here might be helpful: 

```suggestion
Note if an instance returned validation warnings for searchable attributes at the time the instance was stored, those attributes may not be retrievable via search.
```